public/js/app/components/AnalysisTable.jsx

```diff
@@ -31,23 +30,24 @@ const ActiveOrDisabledLink = ({ fileName, linkTitle, storageUri, roundName, vali
 
 // NOTE: uses state from & is hard coded to archive page.
 // So less a 'component' and more a part of a page.
```

Comment:
I think this is an interesting observation. I interpret this to mean that the author was reflecting on that this component resides inside the `components`-folder, but probably won't be used somewhere else than as part of a page. This sounds to me as if the author thinks that components inside the `components`-folder should be reusable by any other component. 
Good to reflect on that and we can keep that in mind for possible future refactorings.